pom.xml

```diff
@@ -1724,119 +966,248 @@
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-gpg-plugin</artifactId>
-                    <version>${version.gpg.plugin}</version>
+                    <artifactId>maven-antrun-plugin</artifactId>
+                    <version>${version.maven.antrun.plugin}</version>
+                </plugin>
```

Comment:
This plugin was used for applying patches to generate the `-orm6`/`-jakarta` artifacts, it can be safely removed (along with the version property).